typescript/raven.d.ts

```diff
@@ -75,6 +75,9 @@ declare module Raven {
         /** By default, Raven captures as many as 100 breadcrumb entries. If you find this too noisy, you can reduce this number by setting maxBreadcrumbs. Note that this number cannot be set higher than the default of 100. */
         maxBreadcrumbs?: number;
 
+        /** A function that allows filtering or mutating breadcrumb payloads. Return false to throw away the breadcrumb. */
+        breadcrumbCallback?: (data: any) => any;
```

Comment:
should this be any->boolean? we don't generally care about correctness with these right? i'm afraid people will return undefined by accident and throw everything away

Reply:
Yeeeeah, it's kinda confusing now. You have to return either payload (an object) or a falsy value to throw it away. This is how we already annotate `dataCallback`, therefore I kept the pattern here.